README.md

```diff
@@ -44,8 +44,10 @@
 ## Quickstart
 
 ```shell
-# with npx
+# with npx and the github repo
 npx react-native init MyApp --template=https://github.com/echobind/react-native-template
+# with npx and the npm package
+npx --ignore-existing react-native init MyApp --template=@echobind/react-native-template
```

Comment:
had to add the `--ignore-existing` because of this weird error?
![image](https://user-images.githubusercontent.com/3806031/67232639-9f33fe80-f3f6-11e9-8c1f-5439d0924a46.png)

Solution credit: https://github.com/react-native-community/react-native-template-typescript/issues/80#issuecomment-536447611